packages/frontend/src/components/FlowRow/DemoFlowModal.tsx

```diff
@@ -31,7 +31,7 @@ export default function DemoFlowModal(props: DemoFlowModalProps): ReactElement {
     ? DEMO_VIDEOS_MAP[demoVideoId]
     : DEMO_VIDEOS_MAP[FORMSG_POSTMAN_TEMPLATE]
 
-  const [showVideoModal, setShowVideoModal] = useState(false)
+  const [showVideoModal, setShowVideoModal] = useState(!isPreCreated ?? false)
```

Comment:
the ?? false is a no-op since !isPreCreated will never be nullish

Reply:
yes boss got it